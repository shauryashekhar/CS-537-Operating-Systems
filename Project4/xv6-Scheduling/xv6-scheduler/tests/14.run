cd src; ../../tester/run-xv6-command.exp CPUS=1 Makefile.test test_14 | grep XV6_SCHEDULER; cd ..
