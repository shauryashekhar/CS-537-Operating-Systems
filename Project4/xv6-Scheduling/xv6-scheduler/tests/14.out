XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 16
XV6_SCHEDULER	 	 level 1 qtail 2
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 0
XV6_SCHEDULER	 	 level 1 qtail 1
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 16
XV6_SCHEDULER	 	 level 1 qtail 2
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 16
XV6_SCHEDULER	 	 level 1 qtail 2
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 32
XV6_SCHEDULER	 	 level 1 qtail 3
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 16
XV6_SCHEDULER	 	 level 1 qtail 2
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 32
XV6_SCHEDULER	 	 level 1 qtail 3
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 32
XV6_SCHEDULER	 	 level 1 qtail 3
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 48
XV6_SCHEDULER	 	 level 1 qtail 4
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 32
XV6_SCHEDULER	 	 level 1 qtail 3
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 48
XV6_SCHEDULER	 	 level 1 qtail 4
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 48
XV6_SCHEDULER	 	 level 1 qtail 4
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 64
XV6_SCHEDULER	 	 level 1 qtail 5
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 48
XV6_SCHEDULER	 	 level 1 qtail 4
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 64
XV6_SCHEDULER	 	 level 1 qtail 5
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 64
XV6_SCHEDULER	 	 level 1 qtail 5
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 80
XV6_SCHEDULER	 	 level 1 qtail 6
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 64
XV6_SCHEDULER	 	 level 1 qtail 5
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 80
XV6_SCHEDULER	 	 level 1 qtail 6
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 80
XV6_SCHEDULER	 	 level 1 qtail 6
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 96
XV6_SCHEDULER	 	 level 1 qtail 7
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 80
XV6_SCHEDULER	 	 level 1 qtail 6
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 1
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 96
XV6_SCHEDULER	 	 level 1 qtail 7
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
XV6_SCHEDULER	 CHILD 2
XV6_SCHEDULER	 	 level 3 ticks used 0
XV6_SCHEDULER	 	 level 3 qtail 0
XV6_SCHEDULER	 	 level 2 ticks used 0
XV6_SCHEDULER	 	 level 2 qtail 0
XV6_SCHEDULER	 	 level 1 ticks used 96
XV6_SCHEDULER	 	 level 1 qtail 7
XV6_SCHEDULER	 	 level 0 ticks used 0
XV6_SCHEDULER	 	 level 0 qtail 0
